DOCS-3766 CI Visibility Pipelines and Tests Index Page Edits (#18580)
DataDog/documentation: content/en/continuous_integration/pipelines/_index.md

{{< img src="ci/ci-pipelines-execution.png" alt="Analytics for a pipeline execution" style="width:100%;">}}

## Use CI pipelines data

> When creating a [dashboards][9] and [notebooks][10], you can use CI pipeline data in your search query, which updates the visualization widget options.

Author: ```suggestion
When creating a [dashboard][9] or a [notebook][10], you can use CI pipeline data in your search query, which updates the visualization widget options.
```

> When creating a [dashboards][9] and [notebooks][10], you can use pipeline execution data in your search query, which updates the visualization widget options.

Author: ```suggestion
When creating a [dashboards][9] and [notebooks][10], you can use CI pipeline data in your search query, which updates the visualization widget options.
```

{{< img src="ci/ci-pipelines-execution.png" alt="Analytics for a pipeline execution" style="width:100%;">}}

## Use CI pipelines data

When creating a [dashboard][8] or a [notebook][9], you can use CI pipeline data in your search query, which updates the visualization widget options. For more information, see the [Dashboards][10] and [Notebooks documentation][11].

> ## Export data to a monitor

Author: ```suggestion
## Alert on pipeline data
```
